src/components/IstioActions/IstioActionsNamespaceDropdown.tsx

```diff
@@ -33,23 +47,45 @@ class IstioActionsNamespaceDropdown extends React.Component<Props, State> {
     history.push('/istio/new/' + type);
   };
 
+  renderTooltip = (key, position, msg, child): any => {
```

Comment:
Same comment, try to use the type that better matches this one (JSX.Element or React.ReactFragment).